src/app/ipv/middleware.js

```diff
@@ -320,6 +320,15 @@ async function renderAttemptRecoveryPage(req, res) {
   });
 }
 
+async function renderStaticPage(req, res) {
```

Comment:
could this take the page id as a parameter? or if not directly could it return a middleware function so we can call it like `staticPageMiddleware(PAGES.PAGE_IPV_IDENTITY_DOCUMENT_TYPES)`